code/web/src/modules/user/ProfileDetails.js

```diff
@@ -28,15 +26,20 @@ class ProfileDetails extends Component {
 			image: '',
 			editMode: false,
 			isLoading: false,
+			editPhotoMode: false
 		}
 	}
 
-	componentDidMount = () => {
+
+
+	componentDidMount = async() => {
```

Comment:
Do we still need to have async await here? I know we were trying it out at some point to see if it would have an effect.